docker_mcp/server.py

```
@@ -42,6 +43,24 @@
 
+def get_data_dir() -> Path:
+    """Get data directory based on environment."""
+    if os.getenv("DOCKER_CONTAINER"):
+        return Path("/app/data")
+    else:
+        # Local development path
+        return Path.home() / ".docker-mcp" / "data"
+
+
+def get_config_dir() -> Path:
+    """Get config directory based on environment.""" 
```

Ruff (0.12.2)

Trailing whitespace

Remove trailing whitespace

(W291)